andy-uq/HomeTrack
# Change storage layer from RavenDB to SQL

- Set up SQL store projects
- Add fluent assertions to test projects
- Make transaction ids a sha1 of immutable values - allows us to make sure we don't import transactions twice
- Use strings for transaction ids
- Refactor general ledger so data changes are controlled by the primary GeneralLedger, not the repository
- Remove RavenDb from solution
